test/python/api/test_normalization.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
File name is not ideal. `test_accept_language_parsing.py` or something along those lines would be better. Or just move the content into the transliterate_locales test below, which then would give us one test file testing all functionality of TransliterateLocales.